### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/markdown/components.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#510] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/markdown/components.html

通过使用vuepress-plugin-components，你可以在你的 Markdown 文件中导入和使用一些组件。 可用组件: ArtPlayer AudioPlayer Badge BiliBili CodePen FontIcon PDF Replit SiteInfo StackBlitz VideoPlayer YouTube 默认情况下，<Badge /> 和 <FontIcon /> 是启用的。 要启用组件，你应该使用组件名称数组设置 plugin.components.components。

https://theme-hope.vuejs.press/zh/guide/markdown/components.html

**Comment by anwen-anyi** [comment]
大佬，问个蠢事儿 哔哩哔哩的怎么设置不自动播放呢？ 我看参数列表里面是没有的QAQ :innocent: :innocent: :innocent:
